presto-clickhouse/src/test/java/com/facebook/presto/plugin/clickhouse/TestClickHouseDistributedQueries.java

```diff
@@ -61,7 +61,7 @@ protected QueryRunner createQueryRunner()
     {
         this.clickhouseServer = new TestingClickHouseServer();
         return createClickHouseQueryRunner(clickhouseServer,
```

Comment:
It looks like it will work. We can refactor to use the other constructor

https://github.com/prestodb/presto/blob/master/presto-clickhouse/src/test/java/com/facebook/presto/plugin/clickhouse/ClickHouseQueryRunner.java#L39C5-L43C6